composites/Plugin/Shared/components/Button.js

```diff
@@ -20,77 +23,33 @@ export function addFocusStyle( component ) {
 			outline: none;
 			border-color: ${ colors.$color_blue };
 			background-color: ${ colors.$color_white };
-			box-shadow: 0 0 3px ${ rgba( colors.$color_blue_dark, .8 ) };
-		}
-	`;
-}
-
-/**
- * Returns a component with applied hover styles.
- *
- * @param {ReactElement} component The original component.
- *
- * @returns {ReactElement} Component with applied hover styles.
- */
-export function addHoverStyle( component ) {
-	return styled( component )`
-		&:hover {
-			background: ${ colors.$color_button_hover };
-			border-color: ${ colors.$color_button_border_hover };
-			color: ${ colors.$color_button_text_hover };
+			box-shadow: 0 0 3px ${ rgba( `${ colors.$color_blue_dark }`, .8 ) };
 		}
 	`;
 }
 
 /**
- * Returns a component with applied active styles.
- *
- * @param {ReactElement} component The original component.
- *
- * @returns {ReactElement} Component with applied active styles.
- */
-export function addActiveStyle( component ) {
-	return styled( component )`
-		&:active {
-			background: ${ colors.$color_button };
-			border-color: ${ colors.$color_button_border_hover };
-			box-shadow: inset 0 2px 5px -3px ${ rgba( colors.$color_button_border_active, 0.5 ) };
-		}
-	`;
-}
-
-/**
- * Returns a component with all button selector styles applied.
- *
- * @param {ReactElement} component The original component.
- *
- * @returns {ReactElement} Component with applied styles.
- */
-export const addButtonStyles = _flow( [ addFocusStyle, addHoverStyle, addActiveStyle ] );
-
-/**
  * Returns a basic styled button.
  *
  * @param {object} props Component props.
  *
  * @returns {ReactElement} Styled button.
  */
-export const BaseButton = addButtonStyles(
+export const BaseButton = addFocusStyle(
 	styled.button`
-		color: ${ colors.$color_button_text };
+		display: inline-block;
+		padding: 8px 10px;
```

Comment:
Please try if 8px looks good as well (because of the 8pt grid, see below). If not, the 10px can stay.